src/Bielu.Examine.ElasticSearch/Services/ElasticsearchService.cs

```diff
@@ -371,27 +371,31 @@ private BulkRequestDescriptor ToElasticSearchDocs(IEnumerable<ValueSet> docs, st
             }
             catch (Exception e)
             {
-#pragma warning disable CA1848
```

Comment:
That is required to disable analyzer to not complain about performance, I have plan to switch it to other one but it is really complicated.

Reply:
ah ok